### PR TITLE
State changes

### PR DESCRIPTION
**Pull request by webrouse**
Part of: https://keboola.atlassian.net/browse/COM-1210

### CODE REVIEW

File: internal/pkg/diff/diff_test.go [hunk 1]
```diff
@@ -763,6 +763,8 @@ func createProjectState(t *testing.T) *state.State {
 	options := state.NewOptions(m, storageApi, schedulerApi, context.Background(), logger)
 	options.LoadLocalState = false
 	options.LoadRemoteState = false
-	s, _ := state.LoadState(options)
+	s, _, localErr, remoteErr := state.LoadState(options)
```
Comment (webrouse) [inline internal/pkg/diff/diff_test.go:766]: `LoadState` nereturnovalo chyby, ale boli na to separatne metody `LocalErrors, RemoteErrors`.
To v minulosti malo zmysel, ale teraz uz nema a v tomto PR to upravujem aby sa chyby spracovavali standardnym sposobom.

File: internal/pkg/state/load_test.go [hunk 2]
```diff
@@ -139,65 +136,6 @@ func TestLoadState(t *testing.T) {
 	assert.Empty(t, utils.SortByName(state.ConfigRows()))
 }
 
-func TestValidateState(t *testing.T) {
```
Comment (webrouse) [inline internal/pkg/state/load_test.go:142]: Iba presunute vid nizsie.

File: internal/pkg/state/local.go [hunk 3]
```diff
@@ -1,20 +0,0 @@
-package state
-
-import (
-	"github.com/keboola/keboola-as-code/internal/pkg/utils"
-)
-
-// loadLocalState - manifest -> local files -> unified model.
-func (s *State) loadLocalState() {
```
Comment (webrouse) [inline internal/pkg/state/local.go:8]: presunute do `state.go`

File: internal/pkg/state/remote.go [hunk 4]
```diff
@@ -1,15 +0,0 @@
-package state
-
-import (
-	"github.com/keboola/keboola-as-code/internal/pkg/utils"
-)
-
-// loadRemoteState - API -> unified model.
-func (s *State) loadRemoteState() {
```
Comment (webrouse) [inline internal/pkg/state/remote.go:8]: presunute do `state.go`

File: internal/pkg/state/load_test.go (hunk 2 above)
Comment (webrouse) [reply internal/pkg/state/load_test.go:142]: vid `validate_test.go`

**Review by JakubMatejka — APPROVED** [review]

**Pull request merged**